package.json

```diff
@@ -49,7 +49,8 @@
     "storybook:build": "build-storybook -c .storybook -o .out --static-dir ./app ./storybook/images",
     "storybook:deploy": "storybook-to-ghpages --existing-output-dir .out --remote storybook --branch master",
     "update-changelog": "./development/auto-changelog.sh",
-    "generate:migration": "./development/generate-migration.sh"
+    "generate:migration": "./development/generate-migration.sh",
+    "postinstall": "npx patch-package"
```

Comment:
This seems unsafe. If we're going to use `patch-package`, it should at least be included as a dev dep and included in the lockfile, so we're not downloading a new `@latest` version each time.